benchmarks/elliptic_integrals_benchmark.cpp

```diff
@@ -18,6 +18,35 @@ using quantities::si::Radian;
 
 namespace numerics {
 
+void BM_EllipticF(benchmark::State& state) {
+  constexpr int size = 20;
+
+  std::mt19937_64 random(42);
+  std::uniform_real_distribution<> distribution_φ(0.0, π / 2);
+  std::uniform_real_distribution<> distribution_n(0.0, 1.0);
```

Comment:
We don't need the n's and the loop at line 40 is probably leading to too efficient caching.

Reply:
Done.